client/stylesheets/_theme.scss

```diff
@@ -5,6 +5,8 @@ $font-family-base: $font-family-sans-serif;
 $link-decoration: none;
 $link-hover-decoration: underline;
 
+$toast-max-width: 380px;
```

Comment:
We need a little more width if we want to actually fit all 4 buttons on a single row. There's a little bit of buffer here because I expect to followup this PR with a fix for #600, which means that "Correct" is also going to get an ellipsis.